.github/workflows/coverage.yml

```diff
@@ -55,18 +55,30 @@ jobs:
         pip install --progress-bar off .[optional]
         pip install --progress-bar off .[integration] --extra-index-url https://download.pytorch.org/whl/cpu
 
+        echo 'import coverage; coverage.process_startup()' > sitecustomize.py
+
     - name: Tests
       env:
         OMP_NUM_THREADS: 1
+        PYTHONPATH: .
+        COVERAGE_PROCESS_START: .coveragerc
+        COVERAGE_COVERAGE: yes
       run: |
-        pytest --cov=optuna --cov-report=xml tests -m "not skip_coverage" \
+        coverage run --source=optuna -m pytest tests -m "not skip_coverage" \
             --ignore tests/integration_tests/test_pytorch_lightning.py
+        coverage combine
+        coverage xml
 
     - name: Multi-node tests
       env:
         OMP_NUM_THREADS: 1
+        PYTHONPATH: .
+        COVERAGE_PROCESS_START: .coveragerc
+        COVERAGE_COVERAGE: yes
```

Comment:
> The purpose of the environment variable is whether or not to measure coverage for the coverage itself. By setting it, tracers that are not recognized by the main tracer of coverage run will also dump results.

Is this documented anywhere? The reference to the document will help for developers in the future. I only found the source code. 😇 

https://github.com/nedbat/coveragepy/blob/65bf33fc03209ffb01bbbc0d900017614645ee7a/coverage/env.py#L119-L120

Reply:
I cannot find any explanation of `COVERAGE_COVERAGE` in [the official doc](`https://coverage.readthedocs.io/en/6.4.1/index.html`) neither. 😇 I think this environment variable is for `coveragepy` developers, and its purpose is shortly explained in the source code here.

https://github.com/nedbat/coveragepy/blob/65bf33fc03209ffb01bbbc0d900017614645ee7a/coverage/control.py#L255-L261

Reply:
I see, thank you! So I'm wondering if it'd be better to leave the comment in the configuration (https://github.com/optuna/optuna/pull/3347/files#r912462028). :pray: